packages/discovery-provider/src/queries/get_users.py

```diff
@@ -39,7 +39,7 @@ def get_users_and_ids():
             # Don't return the user if they have no wallet or handle (user creation did not finish properly on chain)
             if "include_incomplete" not in args or not args["include_incomplete"]:
                 base_query = base_query.filter(
-                    User.is_current == True, User.wallet != None, User.handle_lc != None
```

Comment:
necessary to get guest users by wallet for blockchain user ID association for guest accounts. needed to get emails working for guests.

Reply:
Subtle, nice job